relayer/chains/stacks/clarity/types.go

```diff
@@ -0,0 +1,414 @@
+package clarity
+
+import (
+	"encoding/binary"
+	"encoding/hex"
+	"fmt"
+	"math/big"
+	"sort"
+	"strings"
+)
+
+const (
+	CLARITY_INT_SIZE      = 128
+	CLARITY_INT_BYTE_SIZE = CLARITY_INT_SIZE / 8
+)
+
+var (
+	MaxInt128  = new(big.Int).Sub(new(big.Int).Lsh(big.NewInt(1), 127), big.NewInt(1))
+	MinInt128  = new(big.Int).Neg(new(big.Int).Lsh(big.NewInt(1), 127))
+	MaxUint128 = new(big.Int).Sub(new(big.Int).Lsh(big.NewInt(1), 128), big.NewInt(1))
+)
+
+type ClarityType byte
+
+const (
+	ClarityTypeInt ClarityType = iota
+	ClarityTypeUInt
+	ClarityTypeBuffer
+	ClarityTypeBoolTrue
+	ClarityTypeBoolFalse
+	ClarityTypeStringASCII
+	ClarityTypeStringUTF8
+	ClarityTypeList
+	ClarityTypeTuple
+)
```

Comment:
Isn't the full list? 
  
  Int = 0x00,
  UInt = 0x01,
  Buffer = 0x02,
  BoolTrue = 0x03,
  BoolFalse = 0x04,
  PrincipalStandard = 0x05,
  PrincipalContract = 0x06,
  ResponseOk = 0x07,
  ResponseErr = 0x08,
  OptionalNone = 0x09,
  OptionalSome = 0x0a,
  List = 0x0b,
  Tuple = 0x0c,
  StringASCII = 0x0d,
  StringUTF8 = 0x0e,
}

Reply:
Yeah some will need to be added as I write the transaction builder

Reply:
Unless I can get away with only implementing the strings (that was the original reason I skipped those other ones) but it's looking like it may be necessary to add the rest.

Reply:
I think you need all in the enum at least to get the correct prefixes